lapce-proxy/src/lsp.rs

```diff
@@ -414,8 +454,12 @@ impl LspClient {
         });
     }
 
-    fn process(exec_path: &str) -> Child {
+    fn process(exec_path: &str, binary_args: Vec<String>) -> Child {
```

Comment:
```suggestion
    fn process(exec_path: &str, binary_args: &[String]) -> Child {
```

Oh, I see this is one of your TODOs.